mathics/builtin/scoping.py

```diff
@@ -650,6 +649,9 @@ class BeginPackage(Builtin):
              Protect[System`Private`$ContextPathStack, System`$Packages];
              context
         """,
+        # "BeginPackage[context_String, needs_List]": """
```

Comment:
Uncomment this

Reply:
In that case, let me suggest adding a rule like
```
"BeginPackage[context_String, needs_List]":
"""Print["Current kernel does not support the second argument ",needs, ". Please load them explicitly inside the package"]; BeginPackage[context]"""
```
in a way that the main package be loaded, and helps the user to detect what could go wrong.